Automattic/jetpack
# Contact Form: remove link to editor from old cobebase

Fixes #31774

## Proposed changes:

This applies the changes from #31270 to the old codebase too. Since we do not use that codebase by default since #28791, this should not change anything to the current behavior.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

Not much to test here, as I mentioned above. You could try using the `jetpack_contact_form_use_package` filter to **not** use the package anymore, then add a form to your site, submit a form, and see the email you receive, to check that this works properly.
Or you can check that CI is happy.
